src/commands/create_account.js

```diff
@@ -14,31 +14,55 @@
  *
  */
 import cryptography from '../utils/cryptography';
+import { ValidationError } from '../utils/error';
 import { createCommand } from '../utils/helpers';
 import { createMnemonicPassphrase } from '../utils/mnemonic';
+import commonOptions from '../utils/options';
 
-const description = `Returns a randomly-generated mnemonic passphrase with its corresponding public key and address.
+const description = `Returns a list of randomly-generated mnemonic passphrase with its corresponding public key and address.
 
-	Example: create account
+	Examples:
+	- create account
+	- create account -n 50
 `;
 
-export const actionCreator = () => async () => {
-	const passphrase = createMnemonicPassphrase();
-	const { privateKey, publicKey } = cryptography.getKeys(passphrase);
-	const { address } = cryptography.getAddressFromPublicKey(publicKey);
-
-	return {
-		passphrase,
-		privateKey,
-		publicKey,
-		address,
-	};
+const checkIsInt = number => Number.isInteger(parseInt(number, 10));
+
+const checkIsPositiveInt = number => parseInt(number, 10) > 0;
+
+export const actionCreator = () => async ({ options }) => {
+	const { number = 1 } = options || {};
+
+	if (!checkIsInt(number)) {
+		throw new ValidationError(
+			`${number} doesn't represent a valid Integer corresponding to number of accounts`,
+		);
+	}
+
+	if (!checkIsPositiveInt(number)) {
+		throw new ValidationError('Please provide a number greater than zero');
+	}
+
+	const accountsArray = new Array(number).fill(0).map(() => {
+		const passphrase = createMnemonicPassphrase();
+		const { privateKey, publicKey } = cryptography.getKeys(passphrase);
+		const { address } = cryptography.getAddressFromPublicKey(publicKey);
+		return {
+			passphrase,
+			privateKey,
+			publicKey,
+			address,
+		};
+	});
+
+	return accountsArray;
```

Comment:
This would be a breaking change. I think the normal use case is probably going to be a single account. How about returning an object if the option isn't passed, and a list otherwise?